packages/font/package.json

```diff
@@ -20,8 +20,10 @@
     "ncc-fontkit": "ncc build ./fontkit.js -o dist/fontkit"
   },
   "devDependencies": {
+    "@types/async-retry": "1.4.2",
     "@types/fontkit": "2.0.0",
     "@vercel/ncc": "0.34.0",
+    "async-retry": "1.3.3",
```

Comment:
NOTE: [async-retry](https://www.npmjs.com/package/async-retry) is already in devDeps at [root](https://github.com/vercel/next.js/blob/9b2f29ebaf5add67cec0cac7296f9e64567b3fb4/packages/next/package.json#L200) and [create-next-app](https://github.com/vercel/next.js/blob/9b2f29ebaf5add67cec0cac7296f9e64567b3fb4/packages/create-next-app/package.json#L40)

:link: https://github.com/vercel/next.js/pull/56583#discussion_r1349862707